.github/workflows/build.yml

```diff
@@ -33,12 +33,18 @@ jobs:
     needs: skip-check
     if: ${{ needs.skip-check.outputs.should_skip != 'true' }}
     steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.6
       - uses: actions/checkout@v2
       # see: https://golangci-lint.run/usage/configuration/#config-file
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2.5.2
         with:
           version: v1.42.1
+          skip-go-installation: true
+          skip-pkg-cache: true
```

Comment:
Could you briefly explain this option switch?

Reply:
https://github.com/golangci/golangci-lint-action/blob/master/README.md?plain=1#L62
There is an issue open for this https://github.com/k8gb-io/k8gb/issues/870 I will split the PR into two separate ones to make this cleaner.

Reply:
I've extracted those bits that deals with `golangci-lint` into a standalone [pr](https://github.com/k8gb-io/k8gb/pull/873) and the `skip-pkg-cache` switch actually was not necessary, so good catch. Now the build on this PR is obviously failing and needs to be re-run after the pr 873 is merged, actually whole master is now in troubles, but there were no other PRs going in so it looks ok :D